packages/client/tests/functional/logging/tests.ts

```diff
@@ -7,177 +7,168 @@ import type { Prisma, PrismaClient } from './node_modules/@prisma/client'
 
 declare let newPrismaClient: NewPrismaClient<typeof PrismaClient>
 
-testMatrix.setupTestSuite(
-  (suiteConfig, _suiteMeta, clientMeta) => {
-    let client: PrismaClient<Prisma.PrismaClientOptions, 'query'>
-
-    test('should log queries on a method call', async () => {
-      client = newPrismaClient({
-        log: [
-          {
-            emit: 'event',
-            level: 'query',
-          },
-        ],
-      })
+testMatrix.setupTestSuite((suiteConfig, _suiteMeta, clientMeta) => {
+  let client: PrismaClient<Prisma.PrismaClientOptions, 'query'>
+
+  test('should log queries on a method call', async () => {
+    client = newPrismaClient({
```

Comment:
I observed this test failed against latest changes in [prisma-engines ](https://github.com/prisma/prisma-engines/pull/3505) when it shouldn't.

The response to the /transaction/tx_id/commit endpoint during the test was:

```json
{
   "extensions":{
      "logs":[
         {
            "span_id":"5ad656c7781a5886",
            "name":"COMMIT",
            "level":"query",
            "timestamp":[
               1674129157,
               441746000
            ],
            "attributes":{
               "duration_ms":2.564
            }
         }
      ],
      "traces":[
         
      ]
   }
   ```